home/views.py

```diff
@@ -7,3 +9,17 @@ def about(request):
 
 def landingpage(request):
     return render(request, 'home/landingpage.html')
+
+
+def new_question(request):
+    form = QuestionForm
+    if request.method == 'POST':
+        questForm = QuestionForm(request.POST)
+        if questForm.is_valid():
+            questForm = questForm.save(commit=False)
+            for currProfile in Profile.objects.all():
+                if (currProfile.user == request.user):
+                    questForm.profile = currProfile
+                    break
```

Comment:
I suggest you to replace the for loop to this line: 

questForm.profile=Profile.objects.filter(user=request.user).first()

Reply:
Thank you, I will update

Reply:
This is a good candidate for a class method, but since you've defined the link between the models I wouldn't be surprised if the ORM already generates something like this for you.

Also since you're probably almost always going to need the profile information alongside the user information in your app you may want to create a middleware component that lodes it automaticallt and stores in request just like the user information is stored.

Reply:
So instead of making a class method, you want me to create a middleware component so I can call to request.profile at views file?

Reply:
yep, seems like it would be a useful think to have througout your application if you'll have more then one view that needs information from the profile

Reply:
Got it, ill do it.